### ISSUE TITLE
Accessibility guidelines for Voiceover

### ISSUE THREAD

**Issue by ApoorvKhatreja** [agilebits/onepassword-app-extension#408]
My app includes the 1Password app extension on the login and signup form to enable ease of authentication for our common users.

However, by default, the UIButton that I've configured with assets from 1Password doesn't behave well with VoiceOver, since it doesn't have an `accessibilityLabel` set on it. If I use the phrase "OnePassword" as the accessibility label for the button, Voiceover gets confused and pronounces it as "Oh-nay-password". Using "1Password" works fine and as expected.

Are there any guidelines for this? Additionally are there any recommendations on localizing the voiceover hints to different languages? My app is localized in 11 different languages.

**Comment by PK2702** [comment]
> My app includes the 1Password app extension on the login and signup form to enable ease of authentication for our common users.
> 
> However, by default, the UIButton that I've configured with assets from 1Password doesn't behave well with VoiceOver, since it doesn't have an `accessibilityLabel` set on it. If I use the phrase "OnePassword" as the accessibility label for the button, Voiceover gets confused and pronounces it as "Oh-nay-password". Using "1Password" works fine and as expected.
> 
> Are there any guidelines for this? Additionally are there any recommendations on localizing the voiceover hints to different languages? My app is localized in 11 different languages.

**Comment by rudyrichter** [comment]
In light of the retirement of this extension, I'm closing out any remaining issues and branches.
